dao/src/main/kotlin/tables/shared/IssuesTable.kt

```diff
@@ -83,3 +103,36 @@ class IssueDao(id: EntityID<Long>) : LongEntity(id) {
         worker = worker
     )
 }
+
+/**
+ * Create an [Issue] from the fields of this [ResultRow]. Using this function, issues can be created from query
+ * results that may include other data as well. The fields of the result define the issue; they must appear in the
+ * order as described in [IssueDao.createFromQuery].
+ */
+@Suppress("ComplexCondition")
+fun ResultRow.toIssue(): Issue {
+    // The exposed library seems not to fully support fields on alias queries when unions are used.
+    // Use the field indexes in order to extract the values from the ResultRow instead of the field names.
+    // Disadvantage: More fragility, losing type safety at compile time.
```

Comment:
Another disadvantage is that, if you refactor the `issues` table, e.g adding or removing a column, you'll silently break this function.

I wonder if we could file a solution with column name lookup instead of column index lookup, referencing the properties' name in `IssueDao`. What do you think ?

Reply:
Actually, I did not look deeper into this code. It existed before, and in this PR I just exposed it. It is certainly beneficial to find a better solution here.

Not sure how to proceed here. Can you live with the current solution or should I drop the first commit (it is not required for the other changes)?

Reply:
No problem I misread the diffs and thought you introduced the changes.
Let's leave it like that for the moment then.